### PR TITLE
Change to updated PVs and commented out ones which cannot be found

### LINKED ISSUE #968: update for APS-U era PVs
With the new storage ring from APS-U, verify the devices that connect with the information from the storage ring and beam line front ends.  Include insertion devices.  Check these files:

- [ ] #969
- [x] #970
- [x] #971
- [x] #972
- [x] #973

### LINKED ISSUE #972: aps_machine.py
- https://github.com/BCDA-APS/apstools/blob/ece96632cb919c34c18bc200645189a5632fa576/apstools/devices/aps_machine.py#L20
- https://github.com/BCDA-APS/apstools/blob/ece96632cb919c34c18bc200645189a5632fa576/apstools/devices/aps_machine.py#L38

### PR DESCRIPTION
**Pull request by MDecarabas**
- Closes #972

### CODE REVIEW

File: apstools/devices/aps_machine.py [hunk 1]
```diff
@@ -88,12 +88,18 @@ class ApsMachineParametersDevice(Device):
     # 'Stored Beam',
     # 'Delivered Beam',
     # 'MAINTENANCE')
-    shutter_permit = Component(EpicsSignalRO, "ACIS:ShutterPermit", string=True)
+
+    # shutter_permit = Component(EpicsSignalRO, "ACIS:ShutterPermit", string=True) # Not Found
```
Comment (prjemian) [inline apstools/devices/aps_machine.py:92]: I wonder if I can find this PV (or its replacement) on one of the other screens...

**Comment by prjemian** [comment]
I found a screen with shutter permit PVs for each beam line:

image

Here is `BEAMLINE PER` for 1ID (`S01ID-FEEPS:ShutterPermitWarnM`):
image

**Comment by prjemian** [comment]
I'll keep searching for some that applies to all beam lines.

File: apstools/devices/aps_machine.py (hunk 1 above)
Comment (prjemian) [reply apstools/devices/aps_machine.py:92]: A beamline-independent PV is not obvious.  Another way is to do a recursive grep through the screen files, such as:

    grep -R ShutterPermit . | grep ACIS

Comment (prjemian) [reply apstools/devices/aps_machine.py:92]: ./asdops/screens/SRcomfort.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/SRcomfort.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/SRcomfort.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/SRInj24s.adl:          chan="ACIS:ShutterPermit"
./asdops/screens/SRInjHybrid.adl:               chan="ACIS:ShutterPermit"
./asdops/screens/SRcomfort_2015.adl:            chan="ACIS:ShutterPermit"
./asdops/screens/SRcomfort_2015.adl:            chan="ACIS:ShutterPermit"
./asdops/screens/SRInj324s.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/SRInj24s_P0.adl:               chan="ACIS:ShutterPermit"
./asdops/screens/SRInj324s_P0.adl:              chan="ACIS:ShutterPermit"
./asdops/screens/SRInjHybrid_P0.adl:            chan="ACIS:ShutterPermit"
```

Yet, this PV does not respond to `caget`

**Comment by MDecarabas** [comment]
Is the move making the prefix part of the initialization and then having then we can keep just: `prefix:ShutterPermitWarnM`?

**Comment by prjemian** [comment]
The `prefix` (as you've shown it) is passed in by the user when creating the python object: `obj = TheDevice("prefix:", name="obj")`.  Note the `:` belongs with the prefix.  Internally, Components can then use the part after the `prefix:`, such as `"ShutterPermitWarnM"`

**Comment by prjemian** [comment]
Since the `fine_pitch` and the `flag` have different PV patterns, the *best-fit* prefix was just the IOC.  Also, could not subclass Mirror2 from Mirror1 without some other gymnastics.  This is so simple, it was not worth doing that.

**Pull request merged**